rmf_site_format/src/door.rs

```diff
@@ -186,12 +186,26 @@ impl From<SingleSwingDoor> for DoorType {
 #[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
 pub struct DoubleSwingDoor {
     pub swing: Swing,
+    /// Length of the left door divided by the length of the right door
+    pub left_right_ratio: f32,
```

Comment:
I've always found this ratio to be kind of clunky and unintuitive. It's something we carried over from traffic editor, but I'm not sure it's the best representation of the split in doors.

I wonder if we should do a value `0.0 -> 1.0` representing what percentage of the frame width is for the "left" door, then the right door will just be `1.0 - x`.